elastic/kibana
# [Semantic text] Add doc link for create inference endpoint

Updates add inference endpoint flyout with [doc url](https://www.elastic.co/guide/en/elasticsearch/reference/current/put-inference-api.html) for creating inference endpoint 

Test instruction: 
1. Visit index management page -> index overview page-> index mappings tab
2. Add new field 
3. Select semantic_text field type
4. Click select inference endpoint drop down
5. Click Add new endpoint 
6. in the flyout, confirm "What's this" link redirect to create inference endpoint page [doc url](https://www.elastic.co/guide/en/elasticsearch/reference/current/put-inference-api.html)
7. Also confirm same, changing tab to connect to api
